web/src/components/CreateButton.js

```diff
@@ -10,25 +10,28 @@ const CREATE_TASK = gql`
         id
         name
         complete
+        duedate
+        userId
       }
     }
   }
 `;
 
 const CreateButton = () => {
   const [createTask, { loading }] = useMutation(CREATE_TASK, {
-    refetchQueries: ["GetTasks"],
+    refetchQueries: ['GetTasksAndUsers'],
   });
 
   const onClickCreate = () => {
-    const variables = { input: { name: "New Task" } };
+    // @todo: with auth we could default userId to current user
+    const variables = { input: { name: 'New Task' } };
     createTask({ variables });
   };
 
   return (
-    <Box sx={{ p: 2, display: "flex", justifyContent: "flex-end" }}>
+    <Box sx={{ p: 2, display: 'flex', justifyContent: 'flex-end' }}>
       <Button variant="outlined" disabled={loading} onClick={onClickCreate}>
-        {loading ? "Loading..." : "New Task"}
+        {loading ? 'Loading...' : 'New Task'}
       </Button>
     </Box>
   );
```

Comment:
Haha. We'll need to make sure our editor's play nicely together. No worries.

Reply:
https://media.giphy.com/media/l0IylSajlbPRFxH8Y/giphy.gif